src/Misc/Matrix.jl

```diff
@@ -590,6 +590,10 @@ function lift(a::fmpz_mod_mat)
   return z
 end
 
+function lift(x::gfp_fmpz_mat)
+  return map_entries(lift , x)
+end
+
```

Comment:
@thofma could you please take a look if this is the way to go? Maybe there is a flint function for it?

Reply:
I guess this is fine for now.